samples/ResourceOwnerPasswordFlow/Code2Format.csproj

```diff
@@ -0,0 +1,9 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I wrote that file as the target for https://github.com/dotnet/codeformatter. It requires a csproj and since ASP.NET-5 doesn't have one, this one just includes all the *.cs files.